paimon-common/pom.xml

```diff
@@ -141,6 +141,12 @@ under the License.
             <version>1.0.5</version>
         </dependency>
 
+        <dependency>
```

Comment:
Is it bundled in paimon-format? You should document this.

Reply:
Shaded this to paimon-common